libkbfs/folder_branch_ops.go

```diff
@@ -4593,6 +4619,63 @@ func (fbo *folderBranchOps) unstageAfterFailedResolution(ctx context.Context,
 	return fbo.unstageLocked(ctx, lState)
 }
 
+func (fbo *folderBranchOps) journalBranchChange(newBID BranchID) {
+	if newBID == NullBranchID {
+		return
```

Comment:
We do nothing here because this should only happen for a `PruneBranch`, and that already handles setting the right head, right? Might be worth a comment and/or a log statement.